tooltips/src/main/java/com/tomergoldst/tooltips/ToolTipsManager.java

```diff
@@ -187,6 +187,20 @@ public boolean dismiss(Integer key) {
         return mTipsMap.containsKey(key) && dismiss(mTipsMap.get(key), false);
     }
 
+    /**
+     * Hide and remove all tooltips.
+     */
+    public void dismissAll() {
+        for (Map.Entry<Integer, View> entry : mTipsMap.entrySet()) {
+            View tipView = entry.getValue();
+            int key = entry.getKey();
+            if (tipView != null && isVisible(tipView)) {
+                mTipsMap.remove(key);
```

Comment:
removing an entry here will cause a java.util.ConcurrentModificationException, Since you are trying to modify the Set while still iterating over it.  instead you can just do `mTipsMap.clear()` at the end of the loop

Reply:
The `clear` method performing the same action as `dismissAll`